fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -647,6 +649,10 @@ public String logKey() {
         LUCENE_BLOCK_READS("lucene block reads", false),
         /** Time to write a file references in Lucene's FDBDirectory.*/
         LUCENE_WRITE_FILE_REFERENCE("lucene write file reference" ,false),
+        /** Documents returned from lucene index scans **/
+        LUCENE_RETURNED_DOCUMENTS("lucene returned documents", false),
```

Comment:
I think I might have trouble telling this apart from `LOAD_SCAN_ENTRY`. Maybe `SCANNED` would be clearer than `RETURNED`?

Reply:
I labeled it LUCENE_SCAN_MATCHED_DOCUMENTS since scanned seemed unclear, we are scanning more than just the documents that we are returning from the method.